static/js/components/ProgressWidget.js

```diff
@@ -0,0 +1,87 @@
+// @flow
+import React from 'react';
+import RaisedButton from 'material-ui/RaisedButton';
+
+export default class ProgressWidget extends React.Component {
+
+  props: {
+    actual: number,
+    total:  number
+  };
+
+  // this is temporary, we can remove it once it is integrate into learners dashboard
+  static defaultProps = {
+    actual: 3,
+    total:  5
+  };
+
+  render() {
+    const { actual, total } = this.props;
+
+    return (
+      <div className="progress-widget">
+        <p className="text heading-text">Progress</p>
+        <div className="circular-progress-widget">
+          {this.circularProgressWidget(80, 9, actual, total)}
+        </div>
+        <p className="text text-course-complete">Courses complete</p>
+        <p className="text heading-paragraph">
+          On completion, you can apply for <br/>
+          the Masters Degress Program</p>
+        <div className="apply-master-btn">
+           <RaisedButton
+             disabledBackgroundColor="#8ee0b0"
+             disabledLabelColor="#25b346"
+             label="Apply for Masters"
+             disabled={true} />
+        </div>
+      </div>
+    );
+  }
+
+  circularProgressWidget: Function = (
```

Comment:
could you move this function above `render`? our usual style is to have `render` be the last method declared on a class.